server/tests/models/ticket_spec.js

```diff
@@ -0,0 +1,54 @@
+import chai, { expect, assert } from 'chai';
+import _ from 'lodash';
+import Ticket from '../../models/ticket';
+
+chai.config.includeStack = true;
+
+describe('## Ticket Model', () => {
+  it("should successfully save new ticket", (done) => {
+    let ticket = new Ticket({
+      groupId: '551137c2f9e1fac808a5f572',
+      metadata: {
+        message: 'Something bad!',
+        metrics: {
+          foo: 'bar',
+          jiggawatts: 420
+        }
+      }
+    });
+
+    ticket.save(function (err) {
+      if (err) {
+        assert.fail(err, 'No error');
+      }
+      // saved!
+
+      Ticket.list().then((result) => {
```

Comment:
You could do this, or you could get the saved ticket object in the .save()
Similar to [this](https://github.com/PagerNation/Escalator-API/blob/master/server/tests/models/escalationPolicy.test.js#L22)